data/src/client.rs

```diff
@@ -1169,11 +1276,15 @@ pub struct Topic {
 
 #[derive(Debug, Clone, Copy)]
 pub enum WhoStatus {
-    Requested(Instant),
+    Requested(Instant, Option<u16>),
     Receiving,
     Done(Instant),
 }
 
+fn who_polling_token() -> u16 {
+    9
```

Comment:
I chose it mostly arbitrarily (picked a single digit to make the response messages as short as possible).  It's only being set to help distinguish the `RPL_WHOSPCRPL` from a potential user performed `WHOX`. It's required to be 3 digits or fewer but otherwise does not have a restriction on it.  So we could potentially set it to something else, or have a new token be generated for every poll.